code/modules/atmospherics/machinery/components/unary_devices/cryo.dm

```diff
@@ -170,6 +170,9 @@
 		return
 	var/datum/gas_mixture/air1 = AIR1
 	var/turf/T = get_turf(src)
+	if(occupant && occupant.loc != src)

```

Comment:
This feels kinda hacky, isn't the issue cryo not properly removing the occupant?